### DISCUSSION TITLE
work progress

### DISCUSSION THREAD

**Discussion by listenofficial** [Snowfork/snowbridge#568] [Q&A] [answered]
When I saw that there were 4-6 kinds of USDC in zenlink, I felt that the Polkadot and Kusama ecosystems continued on the Ethereum bridge. I don’t know what stage your work has progressed to, and how long it will take to run a parachain on kusama.

**Comment by musnit** [comment, marked as answer]
The main bottleneck to our launch and on timeline is for the BEEFY Grandpa Bridge Gadget (https://github.com/paritytech/grandpa-bridge-gadget) to be activated on Kusama and Polkadot, rather than any considerations about our parachain or slot. We're ready to launch our parachain as soon as that is activated, and have had our testnet running for a few months now already, but we're blocked on BEEFY.

This blocker applies to both our parachain and any other parachains that want to use our technology, irrespective of whether using our parachain or attempting to do it on their own.

Progress with auditing and activating BEEFY is currently in Parity's hands, not ours. We're hoping to work more closely with them to make it happen sooner over the coming weeks, but they have not been able to give a clear timeline/expected activation date on that yet. We're hoping Kusama activation could happen within the next 2 months.
